### PR TITLE
[image-manipulator] add webp support for Android and iOS

### PR DESCRIPTION
**Pull request by NikitaDudin**
# Why

Added support for converting images to WEBP on Android and iOS.
Removed web-only tag from docs for SaveFormat.WEBP.

# How

On Android used `android.graphics.Bitmap`.
On iOS used `SDImageWebPCoder`.

# Test Plan
- Manually test that the converting work
- Doc review

### CODE REVIEW

**Review by lukmccall — COMMENTED** [review]
Looks good; I've left some suggestions.

File: packages/expo-image-manipulator/ios/ExpoImageManipulator.podspec [hunk 1]
```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |s|
 
   s.dependency 'ExpoModulesCore'
   s.dependency 'EXImageLoader'
+  s.dependency 'SDWebImageWebPCoder', '~> 0.13.0'
```
[the remark this answers is not in the document]
Comment (NikitaDudin) [reply packages/expo-image-manipulator/ios/ExpoImageManipulator.podspec]: Removed version enforcement. It seems to work fine without it.

File: docs/public/static/data/v50.0.0/expo-image-manipulator.json
Comment (lukmccall) [inline docs/public/static/data/v50.0.0/expo-image-manipulator.json:1]: Can you revert that file? It's autogenerated from the code. To achieve the same, you should remove this comment: https://github.com/expo/expo/blob/6ae1b172a1fa88429778cbed518d5c3428c51447/packages/expo-image-manipulator/src/ImageManipulator.types.ts#L97-L99 

Rembert that after you have to rebuild the js by running `yarn build` in the `expo-image-manipulatro` package.

**Review by lukmccall — COMMENTED** [review]
One thing, and it should be ready to be merged ;)

File: docs/public/static/data/v50.0.0/expo-image-manipulator.json
Comment (NikitaDudin) [reply docs/public/static/data/v50.0.0/expo-image-manipulator.json:1]: @lukmccall Thank you! Autogenerated doc file has been reverted. "Platform" specific commentary removed from types file.

**Review by lukmccall — APPROVED** [review]

**Pull request merged**